packages/salesforcedx-vscode-core/src/commands/forceCreateManifest.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { ComponentSet } from '@salesforce/source-deploy-retrieve';
+import * as fs from 'fs';
+import { join } from 'path';
+import { format } from 'util';
+import * as vscode from 'vscode';
+import { nls } from '../messages';
+import { getRootWorkspacePath } from '../util';
+
+export async function forceCreateManifest(
+  sourceUri: vscode.Uri,
+  uris: vscode.Uri[] | undefined
+) {
+  if (!uris || uris.length < 1) {
+    uris = [];
+    uris.push(sourceUri);
+  }
+  const sourcePaths = uris.map(uri => uri.fsPath);
+  if (sourcePaths) {
+    const componentSet = ComponentSet.fromSource(sourcePaths);
+    const inputOptions = {
+        placeHolder: nls.localize('manifest_editor_save_placeholder'),
+        prompt: nls.localize('manifest_editor_save_prompt'),
+        validateInput: text => {
+          return text.includes('.xml') ? 'Remove file extension' : null;
```

Comment:
We don't have "Remove file extension" in other "create" commands. Can we get some feedback from Ananya on this?

Reply:
Yes! Also making a note that this should definitely be localized if we use it.

Reply:
Sure! I scheduled a message to Ananya on this in the team chat for Monday once she's back from PTO.

Reply:
Could we make a common utility for checking for a set of file extensions? 
Something along the lines of 
```
verifyFileExtensions(text: string, invalidExtensions: string[], errorMsg: string): string | null 
```
This could also use `.endsWith` instead of `.includes` so it would allow a filename like `custom.xml.for.contacts`

Reply:
also should we consider casing?  `.XML` also seems like a bummer for case insensitive file systems.

Reply:
I heard back from Ananya here, and she also voted doing the work to strip it out ourselves. So we can keep the messaging, but I'll remove the validation and handle it downstream.

Reply:
@jag-sfdc this has been updated